recipes-devtools/pcie-debug/files/Makefile

```diff
@@ -0,0 +1,9 @@
+#LDFLAGS += -lreadline -lcurses
+
+default: pci_debug
+pci_debug: pci_debug.c
+
+.PHONY : clean
+clean :
+	-rm pci-debug $(objects)
```

Comment:
objects is not set anywhere. The builtin target for creating an executable from a .c file does not generate any objects.

default is also a phony target. Needed by bbclass? Or why add it?

But, as I wondered above. Why not make the changes to the Makefile in the repo?

Reply:
So many questions :wink: But yes it is a good concern and I will make the changes in the repo instead. This is some leftover from testing I guess...